string behavior
int32 behavior_code
geometry_msgs/Point32 target
int32 tower_target
float32 priority
